service/service.go

```diff
@@ -139,6 +141,17 @@ func New(options ...Option) (Owner, error) {
 		metrics.Freeze()
 	}
 
+	if svc.RuntimeMetricsCollector() == nil {
+		var runtimeMetricsConfig metrics.RuntimeConfig
+		err := svc.configProvider.Get("runtimeMetrics").PopulateStruct(&runtimeMetricsConfig)
```

Comment:
we already have `metrics` top-level struct for config. Perhaps we should nest `runtime` under that?